install/helm/agones/templates/crds/_gameserverspecschema.yaml

```diff
@@ -153,6 +153,42 @@ properties:
             type: integer
             title: The initial player capacity of this Game Server
             minimum: 0
+{{- if .featureCountsAndLists }}
+      counts:
+        type: array
+        title: Array of player, room, session, etc. counters
+        nullable: true
+        maxItems: 1000
+        items:
+          type: object
+          properties:
+            name: # counter name
+              type: string
+            count: # current count
+              type: integer
```

Comment:
is this to avoid users of the CRD having to get the size of the array? o/w why do we need a count?

Reply:
Yep, that's all it is. We'll use it for aggregating number of players/rooms/sessions etc. across gameservers, but again that's just to avoid getting the size of the array. Any opinions on if we should leave it in/out?

Reply:
yeah I just realized it was probably a convenience thing. Add it to the validation for sure, though.